stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/MessageListItemViewHolderFactory.kt

```diff
@@ -141,6 +144,12 @@ public open class MessageListItemViewHolderFactory {
         )
     }
 
+    protected fun createGiphyAttachmentViewHolder(
```

Comment:
Comments!

Reply:
+ mention this in CHANGELOG. This is a breaking change